Added information on handling per client rate limit (#2614)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

| **Log per client**                          | The rate limit is based on the [org-wide rate limit](/docs/reference/rate-limits/) values, but the client-specific rate limit violation information is logged as System Log events. |
| **Do nothing**                                | Rate limits aren't enforced at the client-specific level. The rate limit is based on the [org-wide rate limit](/docs/reference/rate-limits/) values. No new or additional System Log events are produced from this feature in this mode. |

#### Per client rate limit

> As of October 12, 2021, the client-based rate limits for the OAuth 2.0 `/authorize` and `/login/login.htm` endpoints are set to **Log per client** mode. This means that if an org's client-based rate limit was previously set to **Do nothing**, the setting is changed to **Log per client** mode.

Reviewer: Do we want to say "..., the default client-based rate limits for... "?

Author: 👍

> As of October 12, 2021, the client-based rate limits for the OAuth 2.0 `/authorize` and `/login/login.htm` endpoints are set to **Log per client** mode. This means that if an org's client-based rate limit was previously set to **Do nothing**, the setting is changed to **Log per client** mode.

Author: 👍

> As of October 12, 2021, the default client-based rate limit for the OAuth 2.0 `/authorize` and `/login/login.htm` endpoints is set to **Log per client** mode. This means that if an org's client-based rate limit was previously set to **Do nothing**, the setting is changed to **Log per client** mode.

Reviewer 2: Lets add a statement that also says if you are already in "enforce and log mode" then it will remain untouched.

> If an org's client-based limit was previously set to **Enforce limit and log per client (recommended)** mode, the setting remains as is. For this mode setting, you need to monitor your syslog events for [`system.client.rate_limit.notification`](/docs/reference/api/event-types/?q=system.client.rate_limit.notification) and [`system.client.concurrency_rate_limit.notification`](/docs/reference/api/event-types/?q=system.client.concurrency_rate_limit.notification) for notifications when a client has a violation. If you see one-time notifications from a handful of different users, then those users may be doing something scripted or automated, and in such cases, no action is required. However, if you see widespread notifications from a large number of users, then you need to troubleshoot and make changes to the application where needed.

Reviewer: ```suggestion
If an org's client-based limit was previously set to **Enforce limit and log per client (recommended)** mode, the setting remains as is. For this mode setting, you need to monitor your syslog events for [`system.client.rate_limit.notification`](/docs/reference/api/event-types/?q=system.client.rate_limit.notification) and [`system.client.concurrency_rate_limit.notification`](/docs/reference/api/event-types/?q=system.client.concurrency_rate_limit.notification) in case of client violations. If you see these one-time notifications from a handful of different users, then those users may be doing something scripted or automated, and in such cases, no action is required. However, if you see widespread notifications from a large number of users, then you need to troubleshoot and make changes to the application where needed.
```

| **Log per client**                          | The rate limit is based on the [org-wide rate limit](/docs/reference/rate-limits/) values, but the client-specific rate limit violation information is logged as System Log events. |
| **Do nothing**                                | Rate limits aren't enforced at the client-specific level. The rate limit is based on the [org-wide rate limit](/docs/reference/rate-limits/) values. No new or additional System Log events are produced from this feature in this mode. |

#### Per client rate limit

As of October 12, 2021, the default client-based rate limit for the OAuth 2.0 `/authorize` and `/login/login.htm` endpoints is elevated to **Log per client** mode. This means that if an org's client-based rate limit was previously set to **Do nothing**, the setting is changed to **Log per client** mode.

> If an org's client-based limit was previously set to **Enforce limit and log per client (recommended)** mode, the setting remains as is. For this mode setting, admins need to monitor System Log events for [`system.client.rate_limit.notification`](/docs/reference/rl-system-log-events/#web-request-rate-limits-client-level) and [`system.client.concurrency_rate_limit.notification`](/docs/reference/rl-system-log-events/#web-request-rate-limits-client-level). If admins don't see any of those events in the System Log, then no action is needed. If admins see these notifications sporadically from just a small number of different users, then those users may be doing something scripted or automated. In such cases, admins can choose to either investigate the issue or choose not to take action. However, if admins see widespread notifications from a large number of users, then it is likely that the application may have an issue, in which case admins need to troubleshoot and make changes to the application where needed.

Reviewer 3: ```suggestion
If an org's client-based limit was previously set to **Enforce limit and log per client (recommended)** mode, the setting remains as is. For this mode setting, admins need to monitor System Log events for [`system.client.rate_limit.notification`](/docs/reference/rl-system-log-events/#web-request-rate-limits-client-level) and [`system.client.concurrency_rate_limit.notification`](/docs/reference/rl-system-log-events/#web-request-rate-limits-client-level). If admins don't see any of those events in the System Log, then no action is needed. If admins see these notifications sporadically from just a small number of different users, then those users may be doing something scripted or automated. In such cases, admins can choose to either investigate the issue or take no action. However, if admins see widespread notifications from a large number of users, then it is likely that the application may have an issue, in which case admins need to troubleshoot and make changes to the application where needed.
```
